packages/zoe/test/unitTests/contracts/test-automaticRefund.js

```diff
@@ -6,41 +6,31 @@ import bundleSource from '@agoric/bundle-source';
 import harden from '@agoric/harden';
 
 import { makeZoe } from '../../../src/zoe';
-import { setup } from '../setupBasicMints';
+import { setup } from '../setupBasicMints2';
```

Comment:
Is this name intended to change?

Reply:
Over time, but not necessarily in this PR. I'm trying to switch to a new way of setting up mints for testing purposes. `setupBasicMints` was all array based, but now that we return an object with issuer, mint, amountMath etc, `setupBasicMints2` uses that representation instead. Rather than going through every single test that relies on setupBasicMints and changing it at once, I've been rewriting as I edit tests. I think we can get rid of setupBasicMints soon.

Reply:
Would you mind adding a TODO here, then?

Reply:
Sure!